docs/docs/spec.md

```diff
@@ -340,9 +340,12 @@ To determine whether a type usage appears in a null-marked scope:
 Iterate over all the declarations that enclose the type usage, starting from the
 innermost.
 
-"Enclosing" is defined as follows: Class members are enclosed by classes, which
-may be enclosed by other class members or classes. Top-level classes are
-enclosed by packages, which may be enclosed by modules.
+"Enclosing" is defined as follows:
+
+-   Class members are enclosed by classes.
+-   Classes may be enclosed by other class members or classes.
+-   Top-level classes are enclosed by packages.
+-   Packages may be enclosed by modules.
 
 > Packages are *not* enclosed by "parent" packages.
```

Comment:
Nor are modules enclosed by parent modules, right?

Reply:
True. I don't know that I've heard people who want it to be otherwise (partially because module usage is still somewhat low and "parent" module naming is probably rare), so I'm less worried about that. But we should probably mention in general that modules aren't enclosed by anything, so I've added that.